test/client/setup.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 The Knative Authors
+Copyright 2020 The Knative Authors
```

Comment:
in the future, this doesnt need to changed
the copyright year is based on when the file is first added.